docs/guides/argo-cd-crossplane.md

```diff
@@ -0,0 +1,104 @@
+This doc will help you understand these requirements.
+ 
+Argo CD synchronizes Kubernetes resource manifests stored in a Git repository
+with those running in a Kubernetes cluster (GitOps). It places a label on
+the running resources to know that it should be tracking them. When it sees
+that label on a running resource, it expects it to match the manifest in the
+repo. Additionally, Argo CD makes use of the Kubernetes Owner Ref field to
+correlate additional resources created by operators as a result of the top-level
+synced resources. Owner Ref enforces a rule that a namespace scoped resource
+cannot be the owner of a cluster scoped resource or a resource in another
+namespace. Because of this, Argo CD cannot correlate relationships that are
+structured in either of those two manners.
+ 
+When Crossplane creates namespace scoped Composite Resource Claims (XRCs), they
+result in cluster scoped Composite Resources (XRs) being created. This is a
+relationship structure that Argo CD cannot correlate. One of the nice features
+in Argo CD is its UI where we can visualize those owner ref relationships and
+gleen information on their state. Because of the inability for a owner ref to be
+defined between XRC and XR, we will not see this visualization for Crossplane
+in the Argo CD UI. If we do not use XRC, and start with XR, we will see the
+visualization. This is a tradeoff that needs to be considered when using the
+two together.
+ 
+Another issue when starting from XRC is introduced because Crossplane
+propagates labels from XRC to XR. In this case, Argo CD synchronizes the XRC
+resource manifest from the repo to the cluster. In doing so, it places its
+tracking label on it. Crossplane then propagates that label to the XR. At this
+point, Argo CD sees the XR with its tracking label, does not see a match in the
+repo, and considers the XR to be an out of sync resource that should be removed.
+If 'auto pruning' is enabled, Argo CD will delete the XR. Crossplane will see
+that the XR is missing and recreate it. This will continue in a loop until
+manual intervention is taken.
+ 
+A final issue (This is not unique to Crossplane) can come into play when we
+manage the Crossplane service itself from Argo CD. When there are late
+initialized resources or fields as a result of an Argo CD sync, it can become
+confused and report these as out of sync. Crossplane will create some late
+initialized RBAC resources and fields that can cause this issue to arise.
+ 
+ 
+### Configuring Argo CD with Crossplane
+ 
+#### 1. When using Argo CD to deploy and manage the Crossplane instance:
+ 
+This is a basic Argo CD Application config that will avoid issues with late
+initialized fields:
+ 
+```yaml
+project: default
+source:
+ repoURL: 'https://charts.upbound.io/stable'
+ targetRevision: 1.6.3-up.1
+ chart: universal-crossplane
+destination:
+ server: 'https://kubernetes.default.svc'
+ namespace: upbound-system

```

Comment:
We should probably use upstream Crossplane rather than UXP on https://crossplane.io docs.

Reply:
@negz Yep, that was on my to-do. I got confused and thought the reviewer made that change. If it's merged already, I'll rePR it with fix

Reply:
Ok, didn't realize I had the additional ref to an Upbound chart. I just removed that portion of the issue entirely because it seems to be a preference type of config. Left with most common issue encountered and all directly Crossplane focused.

Reply:
@negz Looks like I didn't realize this one was still open and doubled up on another doc change. Let me know if it's ok to merge as-is or if I need to break these apart with feature branches.

Reply:
@natereid72 I think you just need to address the two other comment threads below and this should be good to merge. I'll close this one out.